tsconfig.json

```diff
@@ -1,4 +1,4 @@
-{
+https://github.com/kolya-movchan/ts_figures_classes.git{
```

Comment:
why did you do it? keep this file untouched
that causes a linter error and other tests can't run

Reply:
Literally, I don't know how that happened :) 
That was an incident I suppose